include/cuco/detail/static_multimap/static_multimap.inl

```diff
@@ -25,6 +25,280 @@
+                                      alloc,
+                                      stream)},
+    empty_value_sentinel_{empty_value_sentinel}
+{
+}
+
+template <class Key,
+          class T,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+constexpr static_multimap<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::
+  static_multimap(Extent n,
+                  double desired_load_factor,
+                  empty_key<Key> empty_key_sentinel,
+                  empty_value<T> empty_value_sentinel,
+                  KeyEqual const& pred,
+                  ProbingScheme const& probing_scheme,
+                  cuda_thread_scope<Scope>,
+                  Storage,
+                  Allocator const& alloc,
+                  cuda::stream_ref stream)
+  : impl_{std::make_unique<impl_type>(n,
+                                      desired_load_factor,
+                                      cuco::pair{empty_key_sentinel, empty_value_sentinel},
+                                      pred,
+                                      probing_scheme,
+                                      alloc,
+                                      stream)},
+    empty_value_sentinel_{empty_value_sentinel}
+{
+}
+
+template <class Key,
+          class T,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+constexpr static_multimap<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::
+  static_multimap(Extent capacity,
+                  empty_key<Key> empty_key_sentinel,
+                  empty_value<T> empty_value_sentinel,
+                  erased_key<Key> erased_key_sentinel,
+                  KeyEqual const& pred,
+                  ProbingScheme const& probing_scheme,
+                  cuda_thread_scope<Scope>,
+                  Storage,
+                  Allocator const& alloc,
+                  cuda::stream_ref stream)
+  : impl_{std::make_unique<impl_type>(capacity,
+                                      cuco::pair{empty_key_sentinel, empty_value_sentinel},
+                                      erased_key_sentinel,
+                                      pred,
+                                      probing_scheme,
+                                      alloc,
+                                      stream)},
+    empty_value_sentinel_{empty_value_sentinel}
+{
+}
+
+template <class Key,
+          class T,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+void static_multimap<Key, T, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::clear(
+  cuda::stream_ref stream)
+{
+  impl_->clear(stream);
```

Comment:
```suggestion
  this->impl_->clear(stream);
```

Not a hard requirement but I think the RAPIDS best practice guide mentions using `this->` when applicable

Reply:
Ah, we do use `this->` for member functions. For data members with `_` suffix, the intention seems to be clear enough.